subprojects/gradle/cd/src/gradleTest/kotlin/com/avito/ci/steps/LintCheckTest.kt

```diff
@@ -54,8 +54,8 @@ class LintCheckTest {
                                     useImpactAnalysis = false
                                     lint { }
                                     artifacts {
-                                        file("lintReportHtml", "${projectDir.path}/app/build/reports/lint-results-release.html")
-                                        file("lintReportXml", "${projectDir.path}/app/build/reports/lint-results-release.xml")
+                                        file("lintReportHtml", "${projectDir.canonicalPath}/app/build/reports/lint-results-release.html")
```

Comment:
I don't know why on mac os file path is corrupted comparing to Linux
But canonicalization makes path absolute + resolve all `../..` and symbolic links